### ISSUE TITLE
Consider clearly distinguishing external links

### ISSUE THREAD

**Issue by axon-qubes** [QubesOS/qubes-issues#1478]
Should our website clearly distinguish external links (e.g., the way Wikipedia does)? If so, does anyone know the best way to do that given the way the site is set up?

**Comment by adrelanos** [comment]
Axon:
> Should our website clearly distinguish external links (e.g., the way Wikipedia does)?

Yes.

**Comment by axon-qubes** [comment]
@bnvk, any ideas about how best to do this?

**Comment by bnvk** [comment]
I like this idea a lot. I suggest adding the [external-link](http://fontawesome.io/icon/external-link/) icon that's in FontAwesome, which we're using!

**Comment by axon-qubes** [comment]
Right, but is there any way to make that icon appear *automatically* next to external links? I think that's the only way it's going to happen in a useful way. I don't think we can rely on anyone marking up their URLs in source to distinguish external links. Even if we tried to, I have a feeling the results would be very inconsistent (lots of links which are actually external but failed to be manually marked-up that way, resulting in many users being misled, which would arguably be worse than not distinguishing links at all).

**Comment by bnvk** [comment]
>  is there any way to make that icon appear automatically next to external links?

Yes, I hear your point about consistency and not requiring people to tag it. This is ideal, for sure. The easiest way would be to use javascript- i'm not sure if that is "OK" with the team, as a lot of security projects are anti JS. A non-JS way might be with some custom Jekyll module written in Ruby or perhaps a flag in the Jekyll interpreter?

**Comment by axon-qubes** [comment]
Yeah, I think a non-JS way would be preferable. Good ideas.

**Comment by Jeeppler** [comment]
Was there any progress on this issue till now? Any solution considered?

**Comment by andrewdavidwong** [comment]
As far as I know, there has been no additional progress on this. Help is still wanted.
